### PR TITLE
Create readme for graphs

### PR DESCRIPTION
**Pull request by SHaaD94**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/2355)

### CODE REVIEW

**Comment by SHaaD94** [comment]
@REASY could you please help me to fill 3 different modechoice pngs description?

File: src/main/scala/beam/sim/GraphReadmeGenerator.scala [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      |
+      |referenceRealizedModeChoice.png - The same as `realizedModeChoice.png`, but with additional reference mode plot
+      |
+      |replanningCountModeChoice.png - Replanning event count per iteration.
+      |
+      |rideHailRevenue.png - Ride hail revenue in dollars, per iteration.
+      |
+      |rideHailUtilisation.png - Ride hail trip number, distributed with number of passengers, per iteration.
+      |
+      |scorestats.png - Simulation scores per iteration
```
Comment (wrashid) [inline src/main/scala/beam/sim/GraphReadmeGenerator.scala]: Average agent plan utility scores

File: src/main/scala/beam/sim/GraphReadmeGenerator.scala [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      |
+      |referenceModeChoice.png - The same as `modeChoice.png`, but with additional reference modes plot if benchmark file is provided (Beam param `beam.calibration.mode.benchmarkFilePath`)
+      |
+      |referenceRealizedModeChoice.png - The same as `realizedModeChoice.png`, but with additional reference mode plot
+      |
+      |replanningCountModeChoice.png - Replanning event count per iteration.
+      |
+      |rideHailRevenue.png - Ride hail revenue in dollars, per iteration.
+      |
+      |rideHailUtilisation.png - Ride hail trip number, distributed with number of passengers, per iteration.
```
Comment (wrashid) [inline src/main/scala/beam/sim/GraphReadmeGenerator.scala]: Number of ride hail trips by passenger distribution, per iteration

File: src/main/scala/beam/sim/GraphReadmeGenerator.scala [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      |
+      |realizedModeChoice.png - Actual number of transportation chooses, distributed by modes, per iteration. Choosen mode might be not used due to different reasons (for example, ride hail can be too far). It that case person will go to replanning and anothe mode will be choosen.
+      |
+      |referenceModeChoice.png - The same as `modeChoice.png`, but with additional reference modes plot if benchmark file is provided (Beam param `beam.calibration.mode.benchmarkFilePath`)
+      |
+      |referenceRealizedModeChoice.png - The same as `realizedModeChoice.png`, but with additional reference mode plot
+      |
+      |replanningCountModeChoice.png - Replanning event count per iteration.
+      |
+      |rideHailRevenue.png - Ride hail revenue in dollars, per iteration.
```
Comment (wrashid) [inline src/main/scala/beam/sim/GraphReadmeGenerator.scala]: Ride hail revenue generated by all ride hail vehicles, per iteration

File: src/main/scala/beam/sim/GraphReadmeGenerator.scala [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+      |
+      |modeChoice.png - Number of transportation chooses, distributed by modes, per iteration.
+      |
+      |realizedModeChoice.png - Actual number of transportation chooses, distributed by modes, per iteration. Choosen mode might be not used due to different reasons (for example, ride hail can be too far). It that case person will go to replanning and anothe mode will be choosen.
+      |
+      |referenceModeChoice.png - The same as `modeChoice.png`, but with additional reference modes plot if benchmark file is provided (Beam param `beam.calibration.mode.benchmarkFilePath`)
+      |
+      |referenceRealizedModeChoice.png - The same as `realizedModeChoice.png`, but with additional reference mode plot
+      |
+      |replanningCountModeChoice.png - Replanning event count per iteration.
```
Comment (wrashid) [inline src/main/scala/beam/sim/GraphReadmeGenerator.scala]: Replanning event count, per iteration. Replanning events are generated when a person is unable to use a planned mode, e.g. due to shortage of seats on buses or shortage of ride hail in a ridehail_transit trip, at the end of the trip.

Comment (SHaaD94) [reply src/main/scala/beam/sim/GraphReadmeGenerator.scala]: fixed

File: src/main/scala/beam/sim/GraphReadmeGenerator.scala (hunk 3 above)
Comment (SHaaD94) [reply src/main/scala/beam/sim/GraphReadmeGenerator.scala]: fixed

File: src/main/scala/beam/sim/GraphReadmeGenerator.scala (hunk 2 above)
Comment (SHaaD94) [reply src/main/scala/beam/sim/GraphReadmeGenerator.scala]: fixed

File: src/main/scala/beam/sim/GraphReadmeGenerator.scala (hunk 1 above)
Comment (SHaaD94) [reply src/main/scala/beam/sim/GraphReadmeGenerator.scala]: fixed

**Comment by SHaaD94** [comment]
test!

**Pull request merged**